### ISSUE TITLE
laterals too many values to unpack

### ISSUE THREAD

**Issue by Jonasvsl** [nens/threedi-api-qgis-client#81]
* threedi-api-qgis-client version:
* Python version:
* Operating System:

### Description
I start the wizard, with model dw_pacific choose laterals
and upload a csv (in attach of this ticket)

I get this message:
An error has occurred while executing Python code: 

ValueError: not enough values to unpack (expected 4, got 3) 
Traceback (most recent call last):
  File "C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\qgis310/python/plugins\threedi_api_qgis_client\widgets\wizard.py", line 302, in load_csv
    values, filename = self.open_upload_dialog()
  File "C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\qgis310/python/plugins\threedi_api_qgis_client\widgets\wizard.py", line 339, in open_upload_dialog
    for row_id, lat_id, connection_node_id, timeseries in laterals_reader:
ValueError: not enough values to unpack (expected 4, got 3)

Python version: 3.7.0 (v3.7.0:1bf9cc5093, Jun 27 2018, 04:59:51) [MSC v.1914 64 bit (AMD64)] 
QGIS version: 3.10.1-A Coruña A Coruña, ef24c526da 

Python Path:
C:/PROGRA~1/QGIS3~1.10/apps/qgis/./python
C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\qgis310/python
C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\qgis310/python/plugins
C:/PROGRA~1/QGIS3~1.10/apps/qgis/./python/plugins
C:\Program Files\QGIS 3.10\bin\python37.zip
C:\PROGRA~1\QGIS3~1.10\apps\Python37\DLLs
C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib
C:\Program Files\QGIS 3.10\bin
C:\PROGRA~1\QGIS3~1.10\apps\Python37
C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib\site-packages
C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib\site-packages\win32
C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib\site-packages\win32\lib
C:\PROGRA~1\QGIS3~1.10\apps\Python37\lib\site-packages\Pythonwin
C:/Users/jonas.vanschrojenste/AppData/Roaming/QGIS/QGIS3\profiles\qgis310/python
C:\Users\jonas.vanschrojenste\AppData\Roaming\QGIS\QGIS3\profiles\qgis310\python\plugins\threedi_api_qgis_client\deps\PyJWT-1.7.1-py2.py3-none-any.whl
C:\Users\jonas.vanschrojenste\AppData\Roaming\QGIS\QGIS3\profiles\qgis310\python\plugins\threedi_api_qgis_client\deps\threedi_api_client-3.0-py2.py3-none-any.whl

[](url)
### What I Did

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

[laterals_dw_pacific.zip](https://github.com/nens/threedi-api-qgis-client/files/4820410/laterals_dw_pacific.zip)

**Comment by DamnBack** [comment]
@Jonasvsl  - your file have different structure than example file that you provided earlier. It has a 3 columns `id,connection_node_id,timeseries` while example file have 4: `ROWID,id,connection_node_id,timeseries`

**Comment by DamnBack** [comment]
Fixed in the version 2.0.0
